catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -52,7 +52,7 @@
           class="org.codice.ddf.catalog.ui.config.ConfigurationApplication"
           destroy-method="destroy">
         <cm:managed-properties
-                persistent-id="org.codice.ddf.catalog.ui.config"
+                persistent-id="org.codice.ddf.catalog.ui"
```

Comment:
I updated to not change any pids, which causes some `.config` files to need to be named -`config.config` or -`configuration.config` to match the original pid. However, the pid of  `org.codice.ddf.distribution.landing-page.properties` was updated to be `org.codice.ddf.distribution.landingpage.properties` so that the '`-`' does not conflict with the factoryPid .config file naming convention.